packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -37,6 +39,10 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
     showErrorIcon,
     children,
     labelProps,
+    readOnlyText,
+    isReadOnly,
+    inputProps, 
```

Comment:
Don't think you are currently passing inputProps to Field from TextFieldBase, you'll need to pass those